tensorflow_addons/image/median_filter_2D.py

```diff
@@ -0,0 +1,96 @@
+from tensorflow.python.ops import array_ops
+from tensorflow.python.util.tf_export import tf_export
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import math_ops
+from tensorflow.python.framework import dtypes
+from tensorflow.python.ops import image_ops_impl
+from tensorflow.python.ops import script_ops
+from tensorflow.python.ops import control_flow_ops
+from tensorflow.python.ops import gen_math_ops
+from tensorflow.python.framework import tensor_shape
+from tensorflow.python.framework import ops
+from tensorflow.python.ops import gen_array_ops
+from tensorflow.contrib.distributions.python.ops import sample_stats
```

Comment:
shouldn't depend on tensorflow.contrib, where is sample_stats moved ?

Reply:
solved

Reply:
Hmmm I missed this in my first review. @facaiy @karmel thoughts on depending on addons depending tfp?